ssz/simple-serialize.md

```diff
@@ -237,7 +251,8 @@ We now define Merkleization `hash_tree_root(value)` of an object `value` recursi
 * `mix_in_length(merkleize(pack_bits(value), limit=chunk_count(type)), len(value))` if `value` is a bitlist.
 * `merkleize([hash_tree_root(element) for element in value])` if `value` is a vector of composite objects or a container.
 * `mix_in_length(merkleize([hash_tree_root(element) for element in value], limit=chunk_count(type)), len(value))` if `value` is a list of composite objects.
-* `mix_in_type(merkleize(value.value), value.type_index)` if `value` is of union type.
```

Comment:
Was `merkleize` incorrect here? and actually should hve been `htr` all along?

Reply:
Yes, that previous `merkleize` is incorrect. The `value` is the union, and `value.value` is the content of the union, which is just another SSZ object, which can be merkleized in any type-specific way. E.g. if `value.value` was a `List`, `merkleize` is wrong because it would miss the length mix-in.